### ISSUE TITLE
Add Possible Electron GUI

### ISSUE THREAD

**Issue by aadibajpai** [aadibajpai/SwagLyrics-For-Spotify#749]
A tiny GUI might be useful and may be worth looking into as a feature. Something to have in your tray to just pop open and get lyrics in it instead of opening via cmd.

Relevant links:
- https://github.com/SouravJohar/python-app-with-electron-gui

**Comment by GermaVinsmoke** [comment]
Electron apps consume a lot of memory though!

**Comment by aadibajpai** [comment]
@GermaVinsmoke Which is why I say "possible", I mentioned electron because the UIs with it are quite more aesthetic than say one with PyQt. If there's a better option then I'm all ears 🙂

**Comment by GermaVinsmoke** [comment]
Yeah, you're right electron GUIs looks better, I was looking for some better alternatives in Python only (if anything better than PyQT is available or not).
